aiodocker/services.py

```diff
@@ -0,0 +1,93 @@
+import json
+
+
+class DockerServices(object):
+    def __init__(self, docker):
+        self.docker = docker
+
+    async def list(self, filters: dict={}) -> list:
```

Comment:
having mutable structure as dict is not good idea, probable should be 
`filters: Optional[dict]=None` and then `params=filters or {}`